src/main/java/com/wolox/training/models/Users.java

```diff
@@ -0,0 +1,91 @@
+package com.wolox.training.models;
+
+import com.wolox.training.exceptions.BookAlreadyOwnedException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Users {
```

Comment:
Change to "User". And override table name with @Table annotation (you can't use 'user' given it's reserved)